### ISSUE TITLE
Using the same junos_exporter for more targets

### ISSUE THREAD

**Issue by mbarboni** [czerwonk/junos_exporter#59]
Hi, have an environment with about 80 Juniper devices. I'm able to run the exporter if I set up it to retrieve metrics from 1 target but I'm facing issue in running the exporter with 2 or more targets. The problem is the password that is different for every device, so my question is: is there a way to configure the config.yml file in order to have the list of targets and the pw related of each target?
Something like this:

```
targets:
  - 10.10.10.1
     password: "1Password1"
  - 10.10.10.2
     password: "2Password2"
  - 10.10.10.3
     password: "3Password3"

features:
  bgp: true
  ospf: true
  isis: true
  nat: true
  environment: false
  routes: true
  routing_engine: true
  interface_diagnostic: false
  interfaces: true
  l2circuit: true
  storage: true
  fpc: true
```
Please let me know if you have a solution or an alternative to solve my problem.
Thanks in advance!

**Comment by czerwonk** [comment]
Hi,
at the moment this is not possible. But I think I will imeplement a new config type named ``device`` for 1.0. For backward compatiblity all items in ``tagets`` will be mapped to ``device`` with the default password. This should be a reasonable approach. I aim to implement this feature until the end of August.

**Comment by czerwonk** [comment]
Solved in #60
